src/handlers/choosingArticle.js

```diff
@@ -153,17 +154,39 @@ export default async function choosingArticle(params) {
 
     const articleReplies = reorderArticleReplies(GetArticle.articleReplies);
     const summary =
-      '這個訊息有：\n' +
-      `${count.RUMOR || 0} 則回應標成 ❌ 含有不實訊息\n` +
-      `${count.NOT_RUMOR || 0} 則回應標成 ⭕ 含有真實訊息\n` +
-      `${count.OPINIONATED || 0} 則回應標成 💬 含有個人意見\n` +
-      `${count.NOT_ARTICLE || 0} 則回應標成 ⚠️️ 不在查證範圍\n`;
+      t`Volunteer editors has publised several replies to this message.` +
+      '\n\n👨‍👩‍👧‍👦 ' +
+      [
+        count.RUMOR > 0
+          ? t`${count.RUMOR} of them say it ❌ contains misinformation`
```

Comment:
Though pretty annoying, should we apply `ngettext` for verbs here as well?

Reply:
Nope XD